### ISSUE TITLE
Global Navigation needs to be cleaned up

### ISSUE THREAD

**Issue by fowlerbm** [BMFowler/CapStone-Project#1]
The global navigation bar needs to be cleaned up i am suggesting:
- An image banner above the navigation with 100% witdh
- Stretching the navigation bar to 100% width

**Comment by domcull3n** [comment]
My boy coming back with the OG Capstone Project @BMFowler

**Comment by fowlerbm** [comment]
Hey @domcull3n hows it going? I'm currently trying to organize everything so we can get this thing finished.

**Comment by fowlerbm** [comment]
I'm currently going through and making issues for any bugs or things that should be fixed so any of us can hop on and pick off issues when we have the time.

**Comment by domcull3n** [comment]
@BMFowler Good idea man, it'd be a nice side project to casually work on for sure. May take a while but it'd be cool haha.
